tools/benchmark/cmd/mvcc-put.go

```diff
@@ -106,16 +108,25 @@ func mvccPutFunc(cmd *cobra.Command, args []string) {
 	rrc := r.Results()
 
 	rc := r.Run()
-	for i := 0; i < totalNrKeys; i++ {
-		st := time.Now()
-		if txn {
+
+	if txn {
+		for i := 0; i < totalNrKeys; i += nrBatch {
```

Comment:
Why can't the benchmark caller pass in `--total=1000 --txn-ops=10` and expect 10k keys to be written? Working with key counts has more edge cases for the benchmark to get wrong (e.g., this loop won't write the expected number of keys when `totalNrKeys % nrBatch != 0` )